Automattic/jetpack
# ThreatFixerButton: Add tooltips

## Description
Adds tooltips for the various states of the `ThreatFixerButton` component, as per designs.

Something to consider: Does this cover ever potential threat/fixer option?

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1524

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch
* Load storybook with `cd projects/js-packages/storybook && npm run storybook:dev`
* View the Threats Fixer Button and the Threats Data Views component
* Review the various button states and ensure they display and react according to designs
* Review the code and ensure the implementation is correct

## Screenshots
![Screen Shot on 2024-11-06 at 14-52-12](https://github.com/user-attachments/assets/87f1ae39-9a4d-47f8-876e-a3ad1150968f)

![Screen Shot on 2024-11-06 at 14-40-10](https://github.com/user-attachments/assets/62869fe9-e781-4fbf-a89b-d661aabc15b1)

![Screen Shot on 2024-11-06 at 14-40-29](https://github.com/user-attachments/assets/9fd57dc1-3cf6-4c36-93f0-814254d0bf50)

![Screen Shot on 2024-11-06 at 14-40-49](https://github.com/user-attachments/assets/31e6a019-f443-4fa8-b122-a3b41752a3ca)

![Screen Shot on 2024-11-06 at 14-41-15](https://github.com/user-attachments/assets/4b39a5ae-6a6f-46c1-9125-311396eba165)

![Screen Shot on 2024-11-06 at 14-41-38 (1)](https://github.com/user-attachments/assets/988631cb-f8fc-46ec-88d0-7f88347b6807)

![Screen Shot on 2024-11-06 at 14-42-03](https://github.com/user-attachments/assets/5440b5bc-8f5c-4c18-bc0d-4a892a573693)

![Screen Shot on 2024-11-06 at 14-42-25](https://github.com/user-attachments/assets/9ff12de0-e5e2-4b1b-95dc-191c1af243e1)

![Screen Shot on 2024-11-06 at 14-42-55](https://github.com/user-attachments/assets/b3fd5932-7fc9-4461-887b-2484e9f32705)

![Screen Shot on 2024-11-06 at 14-43-17](https://github.com/user-attachments/assets/a16dc45b-579a-4d42-8976-016cbeb5db88)

![Screen Shot on 2024-11-06 at 14-43-35](https://github.com/user-attachments/assets/db1b0a34-f3fd-4047-ad03-f21473161e4d)

![Screen Shot on 2024-11-06 at 14-52-43](https://github.com/user-attachments/assets/25070221-2140-4110-a216-46971118e8f3)

![Screen Shot on 2024-11-06 at 14-44-35](https://github.com/user-attachments/assets/f7e8703e-a85b-407d-b601-fc150ea109b5)
